test/evergreen.yml

```diff
@@ -4099,10 +4099,10 @@ buildvariants:
     - name: ".stress-test-ppc-1"
     - name: ".stress-test-ppc-2"
 
-- name: ubuntu1804-zseries
-  display_name: "~ Ubuntu 18.04 zSeries"
+- name: rhel80-zseries
```

Comment:
Since the underlying OS is rhel83, I'd suggest removing the minor version digit in the name, i.e. `rhel8-zseries`.

Reply:
Yeah, that makes sense.